src/topics/async.md

```diff
@@ -0,0 +1,432 @@
+Asynchronous Programming
+========================
+How we perform more than one simultaneous action in Node.js.
+
+Audience: javascript programmers new to node.js.
+
+
+
+What Is Async Programming
+-------------------------
+Asynchronous programming is any technique in which the result of an operation
+is available outside of the normal synchronous (statement by statement) flow
+of code in a program.
```

Comment:
As a document for beginners, it might be best to lead with a more concrete example of the problem we're trying to solve before introducing the solution.